service/service.go

```diff
@@ -74,24 +78,39 @@ func New(config Config) (*Service, error) {
 
 	var err error
 
-	var k8sClient kubernetes.Interface
+	var restConfig *rest.Config
```

Comment:
This got just copied from the KVM operator. We can clean this later to apply latest structural advances. Right now I do not care.